### DISCUSSION TITLE
Does VRR work via HDMI?

### DISCUSSION THREAD

**Discussion by PolpOnline** [hyprwm/Hyprland#1632] [Q&A]
I have a monitor hooked up via HDMI, and VRR does not seem to work, in Windows it works, so did I misconfigure something?
The only thing I did was to put these lines in my config
```json
misc {
	vfr=true
	vrr=1
}
```

Also, does it conflict with VFR? I tried to enable both and then just one, VRR does not seem to work either way.

I am using an `amdgpu` video card with a LG 27GQ50A monitor.

**Comment by iguanajuice** [comment]
I'm pretty sure the in-kernel amdgpu driver doesn't support AMD's FreeSync but instead supports VESA's adaptive sync, which is compatible with FreeSync displays and such. Unfortunately, adaptive sync is only supported on DisplayPort cables, so you might be SoL when using HDMI.
